UseCase.md

```diff
@@ -1,64 +1,82 @@
 
 ## Merchants
 
-Benefits of using BTCPay as a merchant:
+By choosing BTCPay to process payments, you are:
 
 * Saving money (no fees, no subscriptions)
-* Payments go directly to your wallet (No third-party)
-* Enhanced privacy for you and your customers
-* Easy integration and installation
-* Self-sovereignty 
+* Cutting off the middle-man (Payments go directly to your wallet)
+* Enhancing privacy for you and your customers (no address re-use, no IP leaks to third parties)
+* Saving time (easy integration and installation)
+* Protecting yourself from interference in your business (self-sovereignty) 
 
 ### Online Store
 
-If you're a merchant with an e-commerce business, you quickly can easily [deploy BTCPay](https://docs.btcpayserver.org/deployment) and connect it to your store via [integration plugins](https://docs.btcpayserver.org/integrations/) in just a few clicks. BTCPay checkout is no different to any other payment gateway. Your customer gets an invoice. They pay it by scanning a QR code or by copy-pasting the amount and the address. When their payment is confirmed, you will be notified via your e-commerce CMS; you can ship the item. Take a look at [our demo online store](https://store.demo.btcpayserver.org/).
+If you're a merchant running an e-commerce business, you can easily [deploy BTCPay](https://docs.btcpayserver.org/deployment) and connect it to your store via [integration plugins](https://docs.btcpayserver.org/integrations/) in just a few clicks. 
+
+BTCPay checkout is no different than any other payment gateway: your customer gets an invoice, that they pay by scanning a QR code or by copy-pasting the amount and the address in their own Bitcoin wallet. When their payment is confirmed, you will be notified via your e-commerce CMS; you can ship the item. Take a look at [our demo online store](https://store.demo.btcpayserver.org/).
 
 ### Physical Store
 
-For brick and mortar stores, BTCPay has a [web-based Point of Sale](https://mainnet.demo.btcpayserver.org/apps/87kj5yKay8mB4UUZcJhZH5TqDKMD3CznjwLjiu1oYZXe/pos) (POS) App which can be customized. Similarly to the online store, your customer is presented with an invoice which needs to be paid. You can create a watch-only wallet on your phone to be notified of the payments through the POS, without the need of any additional software. There are plans on developing the POS mobile app which would make the process even smoother.
+For brick and mortar stores, BTCPay has a [web-based Point of Sale](https://mainnet.demo.btcpayserver.org/apps/87kj5yKay8mB4UUZcJhZH5TqDKMD3CznjwLjiu1oYZXe/pos) (POS) App which can be customized. Similarly to the online store, your customer is presented with an invoice that he can pay on the spot. You can create a watch-only wallet on your phone to be notified of the payments through the POS, without the need of any additional software. There are plans to develop the POS mobile app which would make the process even smoother.
+
+### Lightning Network payments
```

Comment:
This has to be a separate H2 headline since it's for both users who want to experiment and use LN and merchants. Do not mix these sections.
>